exonum-java-binding/integration-tests/pom.xml

```diff
@@ -61,10 +63,29 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
 
+      <plugin><!--Do not create Jar-->
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-jar</id>
+            <phase/>
```

Comment:
https://maven.apache.org/plugins/maven-jar-plugin/jar-mojo.html#skipIfEmpty is more appropriate